include/libp2p/multi/hash_type.hpp

```diff
@@ -7,6 +7,10 @@
 #define LIBP2P_HASH_TYPE_HPP
 
 namespace libp2p::multi {
+  /// TODO(Harrm) Hash types are a part of multicodec table, it would be good to
```

Comment:
pls add a task for that and specify it or implement and remove TODO.
```suggestion
  /// TODO(Harrm): PRE-*** Hash types are a part of multicodec table, it would be good to
```